cmd/inspector/main.go

```diff
@@ -103,6 +107,11 @@ var (
 		Short: "dump all nodes in the routing table",
 		RunE:  DumpNodes,
 	}
+	drawTableCmd = &cobra.Command{
+		Use:   "routing-graph (<node_id>)",
+		Short: "Create a dot graph of the routing table saved by default as routing-graph.dot",
```

Comment:
Specify the defaults when defining the flag, the name will automatically pop up.